LiquidBounce/settings/cubecraft

```diff
@@ -158,4 +158,15 @@ Velocity toggle true
 NoSlow toggle true
 Sptint toggle true
 AutoArmor toggle true
+InventoryMove toggle true
+ChestStealer toggle true
+InventoryCleaner toggle true
+
 targetPlayer true
+
+chat &a&lCubeCraft Config /// 09/06/2020
+chat &bBefore loading, execute .panic nonrender, then load again!
+chat &b&lSCRIPTS
+chat &bCheck out Float Script: https://forum.ccbluex.net/thread.php?id=3041
+chat &bCheck out Speed Script: https://forum.ccbluex.net/thread.php?id=3039
+chat &bCheck out CubeCore (all in one cubecraft script): https://forum.ccbluex.net/thread.php?id=1537
```

Comment:
I am not sure if those recommendations are really ok since those scripts aren't made by us

Reply:
it's fine if you want to remove that but since liquidbounce doesn't have any of that stuff in the scripts i think it would be good for the user to try, since they all work